### DISCUSSION TITLE
How to set "dateFormat"

### DISCUSSION THREAD

**Discussion by fukugit** [chipzoller/hugo-clarity#129] [Q&A] [answered]
Hi, 
Thank you for providing good theme. I have some question.

I just tried to set the "dateFormat" in the ```config.toml``` like the below, but date format seems not to be changed.
```
[params]
  dateFormat = "yyyy-mm-dd"
```

So, I have two questions. 
1. How do I set the  "dateFormat" in the ```config.toml``` ?
2. Which part will change after changing correctly?

**Comment by onweru** [comment, marked as answer]
@fukugit, you need to follow [this approach](https://gohugo.io/functions/dateformat/#readout). `yyyy-mm-dd` wont work.

**Comment by fukugit** [comment]
@onweru I just understood. It doesn't work on the 'config.toml'.
